### ISSUE TITLE
Warning for fail in HA version 2023.3 after update HA 2023.2

### ISSUE THREAD

**Issue by smarthomefamilyverrips** [Vaskivskyi/ha-asusrouter#506]
### The problem

@ FYI after update to HA 2023.2 I get this warning

Detected integration that called async_setup_platforms instead of awaiting async_forward_entry_setups; this will fail in version 2023.3. Please report issue to the custom integration author for asusrouter using this method at custom_components/asusrouter/init.py, line 48: hass.config_entries.async_setup_platforms(entry, PLATFORMS)

### Your device model

ASUS XT-12

### Firmware type

Stock

### Firmware version

3.0.0.4.388_22127

### Integration version with the issue

0.18.3

### Method of the integration installation

HACS

### What version and type of Home Assistant installation do you use

2023.2

### Is there anything useful in the logs?

```txt
Detected integration that called async_setup_platforms instead of awaiting async_forward_entry_setups; this will fail in version 2023.3. Please report issue to the custom integration author for asusrouter using this method at custom_components/asusrouter/init.py, line 48: hass.config_entries.async_setup_platforms(entry, PLATFORMS)
```

### Additional information

None

**Comment by Vaskivskyi** [comment]
Hey. Thanks for your report. This will be fixed as soon as possible

**Comment by Vaskivskyi** [comment]
Hello. The issue is fixed in the new 0.18.4 version

**Comment by smarthomefamilyverrips** [comment]
Great thanks @Vaskivskyi will update as soon get notification update is available 👍🏻👍🏻👍🏻
